CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - PR #3690 Add bools_to_mask
 - PR #3683 Added support for multiple delimiters in `nvtext.token_count()`
 - PR #3792 Adding is_nan and is_notnan
+- PR #3861 Added Series.sum feature for String
```

Comment:
I've retargeted this PR to branch-0.13, could you move this to the proper changelog section?